examples/tf/EMI-RNN/00_emi_lstm_example.ipynb

```diff
@@ -120,10 +119,10 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "x_train shape is: (6294, 6, 48, 9)\n",
-      "y_train shape is: (6294, 6, 6)\n",
-      "x_test shape is: (1058, 6, 48, 9)\n",
-      "y_test shape is: (1058, 6, 6)\n"
+      "x_train shape is: (6409, 6, 48, 9)\n",
```

Comment:
@pushkalkatara Why did the x_train shape change?

Reply:
@harsha-simhadri Due to [data generation](https://github.com/microsoft/EdgeML/blob/cb8e8eb719241bf2803a905c4f10c90b2b200176/tf/examples/EMI-RNN/helpermethods.py#L150) script, the shapes on each processing is varying. i.e
```
Processing data
Extracting features
('subinstanceLen', 48)
('subinstanceStride', 16)
('sourceDir', '/home/pushkalkatara/mr/EdgeML/examples/tf/EMI-RNN/HAR//RAW/')
('outDir', '/home/pushkalkatara/mr/EdgeML/examples/tf/EMI-RNN/HAR//48_16/')
Num train 6339
Num test 2947
Num val 1013
Done
```
```
Processing data
Extracting features
('subinstanceLen', 48)
('subinstanceStride', 16)
('sourceDir', '/home/pushkalkatara/mr/EdgeML/examples/tf/EMI-RNN/HAR//RAW/')
('outDir', '/home/pushkalkatara/mr/EdgeML/examples/tf/EMI-RNN/HAR//48_16/')
x_train 6335
Num train 6335
Num test 2947
Num val 1017
Done

```